jme3-core/src/plugins/java/com/jme3/material/plugins/J3MLoader.java

```diff
@@ -970,4 +970,4 @@ public void applyToTexture(final Texture texture) {
             textureOption.applyToTexture(value, texture);
         }
     }
-}
+}
```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR